### ISSUE TITLE
Add support for injection token for MatDatepickerToggleIcon

### ISSUE THREAD

**Issue by Chan4077** [angular/material2#15639]
#### Please describe the feature you would like to request.

Provide support for using an injection token for specifying the icon of datepicker toggles.

#### What is the use-case or motivation for this proposal?

Currently, I am migrating my icons to use the new outlined Material Icons (`https://fonts.googleapis.com/css?family=Material+Icons+Outlined`) in my app. However, I have to change the icons by specifying the `matDatePickerToggleIcon` to use for all datepicker toggles.

Having an easy way of specifying the toggle icon to be used for all of the datepicker toggles would be useful.

#### Is there anything else we should know?

NIL

**Comment by crisbeto** [comment]
I don't think we can do this with an injection token, because the `matDatePickerToggleIcon` is passed in as an element.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
